evalml/tests/pipeline_tests/regression_pipeline_tests/test_timeseries_baseline_regression.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+from evalml.pipelines import TimeSeriesBaselineRegressionPipeline
+
+
+def test_time_series_baseline(ts_data):
+    X, y = ts_data
+
+    clf = TimeSeriesBaselineRegressionPipeline(parameters={"pipeline": {"gap": 1, "max_delay": 1}})
+    clf.fit(X, y)
+
+    np.testing.assert_allclose(clf.predict(X, y), y)
+
+
+def test_time_series_baseline_no_X(ts_data):
```

Comment:
Can you please add a test that verifies that the predictions and targets are offset by the right amount in `score` for both the `gap=0` and `gap > 0` cases? This can be similar to `test_score_drop_nans` in `test_time_series_pipeline`.